packages/fxa-payments-server/src/routes/Checkout/index.tsx

```diff
@@ -175,6 +173,7 @@ export const Checkout = ({
           email: validEmail,
           clientId: config.servers.oauth.clientId,
           customer: null,
+          profile: profile || null,
```

Comment:
Same goes here as for `PaypalButton` in terms of passing down `checkoutType` instead of `profile`.

In the case of the `Checkout` route (unlike for the `PaypalButton`), we know `checkoutType` is `'without-account'`.

Reply:
👍